tests/test_international_days.py

```diff
@@ -0,0 +1,31 @@
+from datetime import date
```

Comment:
לדעתי כדאי גם להוסיף בדיקה של הHTML אם הדבר הזה בעצם צריך להגיע לתצוגה

Reply:
why would it be in the fronted?

Reply:
i ment, if you added element to the html template, then you should test it's working on the html page as well. your current tests are for the backend only.

Reply:
My tests are for the backend